README.md

```diff
@@ -4,8 +4,55 @@ This is the API portion of [OpenTelemetry](https://opentelemetry.io/) for Erlang
 
 This is a library, it does not start any processes, and should be the only OpenTelemetry dependency of Erlang/Elixir applications. 
 
-The end user of your application can then choose to either include the [OpenTelemetry implementation](https://github.com/open-telemetry/opentelemetry-erlang) application. If the other application is not in the final release the OpenTelemetry instrumentation will all be noops. This means no processes started, no ETS tables created and nothing added to the process dictionary.
+The end user of your application can then choose to include the [OpenTelemetry implementation](https://github.com/open-telemetry/opentelemetry-erlang) application. If the implementation application is not in the final release the OpenTelemetry instrumentation will all be noops. This means no processes started, no ETS tables created and nothing added to the process dictionary.
 
 This separation is done so you should feel comfortable instrumenting your Erlang/Elixir application with OpenTelemetry and not worry that a complicated dependency is being forced on your users.
 
+## Use
+
+When instrumenting an application to be used as a dependency of other projects it is best practice to register a `Tracer` with a name and a version using the application's name and version.
+
+If it is a runnable application then this registration should happen in `start/2`, example below is adding `Tracer` registration to the Postgres library [pgo](https://github.com/erleans/pgo):
+
+``` erlang
+start(_StartType, _StartArgs) ->
+...
+    {ok, Vsn} = application:get_key(pgo, vsn),
+    opentelemetry:register_tracer(pgo, Vsn),
+...
```

Comment:
any issue in registering more than once? Curious what happens with a live code upgrade

Reply:
There isn't, the only catch is the performance penalty of registration. So I'll make that clear.